core/src/test/scala/io/finch/EndpointSpec.scala

```diff
@@ -438,6 +448,11 @@ class EndpointSpec extends FinchSpec {
         def apply[A](fa: IO[A]): WriterT[IO, List[String], A] = WriterT.liftF(fa)
       }
 
+      implicit val dispatcherW: Dispatcher[W] = new Dispatcher[W] {
```

Comment:
Same: there might be already an instance of `Async` for Writer so we can summon Dispatcher.